python_practice/calculator/simple_calc.py

```diff
@@ -0,0 +1,123 @@
+"""
+This program will be a simple calculator (add, subtract, divide, multiply)
+"""
+
+
+def add(
+        x: int,
+        y: int,
+) -> int:
+    """
+    This function will be the add operator
+
+    :param x: x will act as the first addend
+    :param y: y will act as the second addend
+    :return int: the sum will be an integer
+    """
+
+    total = x + y
```

Comment:
If a variable is only used once, that is a good identifier that you don't need it at all. Simply return x + y